cli/flox-core/src/activations.rs

```diff
@@ -21,15 +21,10 @@ type Error = anyhow::Error;
 /// Incrementing this will require existing activations to exit.
 const LATEST_VERSION: u8 = 1;
 
-#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
 pub struct UncheckedVersion(u8);
-#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
 pub struct CheckedVersion(u8);
-impl Default for CheckedVersion {
-    fn default() -> Self {
-        Self(LATEST_VERSION)
-    }
-}
```

Comment:
I think this makes the default checked version `0`?

Reply:
I thought we didn't need it now that `check_version` always upgrades but it caused all of the tests to fail because it was initially written with the wrong version and then upgraded. Reverted in an additional commit that will be rebased in after review.